index.html

```diff
@@ -246,23 +246,23 @@ <h3 class="mainText CreteRoundfont">About me</h3>
           type="text"
           maxlength="30"
           placeholder="Full Name"
-        />
+        required/>
 
         <input
           class="no-outline"
           type="email"
           name="email"
           placeholder="Email Address"
-        />
+        required/>
 
         <textarea
           name="message"
           maxlength="500"
           rows="7"
           cols="70"
           class="formTxtArea"
-        >
-        Enter text here.</textarea
+          placeholder="Enter text here."
+        required></textarea
         >
         <button type="button" class="PrjBTN SePrjBTN Interfont">
```

Comment:
Your form is currently not making a post request or submitting, try changing the type button to type submit, It probably might fix the issue.